Makefile

```diff
@@ -103,20 +103,59 @@ bindings-examples-format: ## Format all bindings examples
 
 # Build ffi crate and detect platform
 define build_binding
-cargo build -p iota-sdk-ffi --lib --release; \
-case "$$(uname -s)" in \
-	Darwin)   LIB_EXT=".dylib" ;; \
-	Linux)    LIB_EXT=".so" ;; \
-	MINGW*|MSYS*|CYGWIN*|Windows_NT) LIB_EXT=".dll" ;; \
-	*)        echo "Unsupported platform"; exit 1 ;; \
-esac;
+if [ -n "$$TARGET" ]; then \
```

Comment:
I don't know if I would add all that stuff here tbh. It's a lot of noise and is actually only usable through the CI because you can't properly cross-compile.